src/test/java/com/epam/tc/hw4/steps/ActionStep.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.tc.hw4.steps;
+
+import com.epam.pages.DifferentElementsPage;
+import com.epam.pages.MainPage;
+import io.qameta.allure.Step;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ActionStep extends AbstractStep {
+
+    public ActionStep(WebDriver driver) {
+        super(driver);
+    }
+
+    @Step("Open test site by URL")
+    public void openWebSite(String website) {
+        webDriver.get(website);
+    }
+
+    @Step("Perform login")
+    public void performLogin(String user, String password) {
+        mainPage = new MainPage(webDriver);
```

Comment:
That's what we were talking about during our call. Please, place it in your base abstract class.